sdk/src/transaction.rs

```diff
@@ -208,12 +251,92 @@ impl Transaction {
             .for_each(|tx| hasher.hash(&tx.signatures[0].as_ref()));
         hasher.result()
     }
+
+    pub fn serialized_size(tx: &Transaction) -> u64 {
```

Comment:
Can this be a method on Transaction? So that parameter would be `&self`?

Reply:
Done.